### ISSUE TITLE
Error using solid-auto-animate in a pnpm workspace

### ISSUE THREAD

**Issue by niclas-j** [lxsmnsyc/solid-auto-animate#2]
Installing `solid-auto-animate` in a workspace and importing it in a component results in an error resolving the @formkit/auto-animate package.

`Error: Failed to resolve entry for package "@formkit/auto-animate". The package may have incorrect main/module/exports specified in its package.json: No known conditions for "." entry in "@formkit/auto-animate" package`

**Comment by lxsmnsyc** [comment]
Is `@formkit/auto-animate` successfully installed by any chance? You can confirm this either through the root node_modules or through the lockfile

**Comment by niclas-j** [comment]
@lxsmnsyc Yes it is, I also installed it in the root mono repo package just in case that some referencing there could be the reason for it to break.

```
import test from "@formkit/auto-animate";
import { autoAnimate } from "solid-auto-animate";
```

just importing `test`would not break anything, as soon as I import autoAnimate the error occurs.
